plugins/notifications-backend/src/service/db.ts

```diff
@@ -24,7 +24,7 @@ export async function initDB(dbConfig: Config): Promise<Knex<any, any>> {
     await dbClient.schema.createTable('messages', table => {
       table.uuid('id').primary().notNullable().defaultTo(dbClient.fn.uuid());
       table.string('origin').notNullable();
-      table.timestamp('created').defaultTo(dbClient.fn.now()).index();
+      table.string('created').index(); // ISO 8601 date-time string
```

Comment:
without knex i would use the DB types. but with knex and trying to support multiple DBs I don't see what choice we have.
i prefer using a number instead of string. I mean unix timestamp. store the UTC value as number. if you're ok with it I will change it.

Reply:
although...https://www.javascript-coder.com/strings/compare-two-date-strings-javascript/#:~:text=ISO%208601%20date%20strings%20are,%2DDDTHH%3Amm%3Ass.

Reply:
I am ok with numeric values as well.

Not sure I understand what you are referring to regarding the Date comparison in JS.